src/main/resources/interface-specification.yml

```diff
@@ -773,8 +773,9 @@ components:
         expirationDate:
           type: string
           format: date-time
-      required:
-        - renewal    
+        extensionDate:
```

Comment:
if requirements have not been changed, `extensionDate` should be a field calculated only in the backend.
I guess it has been added as request parameter in order to allow the job to update it, right?

Thinking aloud: this could potentially allow another caller (BFF or API GW) to change the field as well, which at the moment is undesired.

What if, instead passing this value to the `updateVerifiedAttribute`, we add a new endpoint in the process that recalculate this date for the given attribute?
The business logic would be confined in the process and no one would be allowed to directly change the value.

cc @beetlecrunch, maybe you had different requirements